cnf/repo/org.osgi.impl.service.remoteserviceadmin/org.osgi.impl.service.remoteserviceadmin-7.0.0.lib

```diff
@@ -1,15 +1,17 @@
 org.osgi.service.remoteserviceadmin; version=latest
 org.osgi.service.cm; version=latest
 org.osgi.impl.service.configurator;version=latest
+org.osgi.impl.service.jaxrs;version=latest
```

Comment:
Thie `org.osgi.impl.service.remoteserviceadmin-7.0.0.lib` file should be renamed to end in `-8.0.0.lib`.